lms/static/scripts/postmessage_json_rpc/client/random.js

```diff
@@ -6,13 +6,12 @@ function byteToHex(val) {
 /**
  * Generate a random hex string of `len` chars.
  *
- * @param {number} - An even-numbered length string to generate.
+ * @param {number} len - An even-numbered length string to generate.
  * @return {string}
  */
 function generateHexString(len) {
-  const crypto = window.crypto || window.msCrypto; /* IE 11 */
   const bytes = new Uint8Array(len / 2);
-  crypto.getRandomValues(bytes);
+  window.crypto.getRandomValues(bytes);
```

Comment:
The LMS frontend has never supported IE 11, so I've simply removed that code path here.